modules/ROOT/pages/footnotes.adoc

```diff
@@ -0,0 +1,135 @@
+= Footnotes plugin
+:navtitle: Footnotes
+:description: Insert footnotes into the {productname} editor
+:keywords: footnotes, footnote, insertfootnotes, insert, footnotesupdate, mceInsertFootnote, mceUpdateFootnotes
+:pluginname: Footnotes
+:plugincode: footnotes
+:pluginminimumplan: tierthree
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+The {pluginname} plugin allows the user to insert a footnote into the {productname} editor.
+
+Each time a footnote is added, a superscripted footnote number is placed at the current insertion point, or at the end of the current selection.
+
+The plugin then adds a new line to the footnotes section located at the bottom of the editor and moves the insertion point to this new line. This allows for the immediate creation of the contents of the footnote entry related to the newly-created footnote
+
+== Interactive Example
+
+liveDemo::footnotes[]
+
+include::partial$misc/purchase-premium-plugins.adoc[]
+
+== Basic setup
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'footnotes',
+  toolbar: 'footnotes',
+});
+----
+
+The footnotes section, which is placed at the bottom of the {productname} editor, is created when the first footnote is added.
+
+Footnotes can be inserted using the _Insert footnote_ button, located on the toolbar, or by selecting the _Insert > Insert Footnote_ command from the menu bar, or by using the `+mceInsertFootnote+` command.
+
+The footnotes section is automatically updated whenever a footnote is inserted, reordered, or removed within the {productname} editor. The footnotes section is automatically removed if all footnotes are removed.
+
+For example, adding a new footnote between already extant footnotes ^*1*^ and ^*2*^ results in the new footnote becoming footnote ^*2*^ and the previous footnote ^*2*^ automatically becoming footnote ^*3*^.
+
+Superscripted footnote numbers can also be dragged to a different position in the document. Footnote numbers and their relationship with the respective footnote entry are automatically updated when they are dragged to a different position.
+
+Each superscripted footnote number links to its respective footnote entry, and each footnote entry is prefixed with a caret — *^* — that links back to its respective superscripted footnote number. This allows for navigation between the editor content and the footnotes section.
+
+If the TinyMCE document is large enough to produce a scroll bar, clicking the caret prefixing a footnote entry scrolls the document back to the line containing said entry’s superscripted footnote number.
+
+NOTE: Although the viewport scrolls back to the line containing the footnote number, the insertion point is not changed. Users must move the insertion point by, for example, clicking the I-beam pointer somewhere in the now-scrolled document.
+
+
+=== CSS
+
+//// explanatory text here.
```

Comment:
This seems to be missing...

- [ ] Fill in missing information

Reply:
Also worth noting this is current rendering as verbatim text... it's probably because `////` is for a block comment but it's being used as an inline comment:
![image](https://user-images.githubusercontent.com/1575550/191658588-ddf976f9-7779-4ff2-8a88-d1dfd3194651.png)

Reply:
The section has been filled out now

Reply:
This section isn't quite correct... you need this when extracting content as well so it's something you should always add. See https://www.tiny.cloud/docs/tinymce/6/checklist/#css

Reply:
I did add this paragraph for that `To make footnotes display as it does in {productname}, be sure to add the CSS to the webpage.` Although maybe that isn't clear enough and my attempt in trying to avoid pronouns might have made it confusing.

Maybe it should be something like: `To make footnotes display correctly outside of {productname}, be sure to add the CSS to the published webpage.`

Reply:
Right, I read it as something a bit different sorry. The new version makes that clear 👍

Reply:
I am now using the new wording. Thanks